src/linux/timerfd.rs

```diff
@@ -71,19 +71,15 @@ impl TimerFd {
         {
             spec.it_value.tv_sec = dur.as_secs() as libc::time_t;
         }
-        // nsec always fits in i32 because subsec_nanos is defined to be less than one billion.
-        let nsec = dur.subsec_nanos() as i32;
-        spec.it_value.tv_nsec = libc::c_long::from(nsec);
+        spec.it_value.tv_nsec = dur.subsec_nanos() as _;
```

Comment:
Personal preference but I find `as _` to be quite unreadable.

```suggestion
        spec.it_value.tv_nsec = dur.subsec_nanos().try_into().unwrap();
```